app/scripts/components/dashboard/waterpoints.jsx

```diff
@@ -33,20 +36,19 @@ const WaterPoints = React.createClass({
               className="leaflet-map">
             <TileLayer url="//{s}.tile.openstreetmap.org/{z}/{x}/{y}.png" />
             {this.state.waterpoints.map(waterpoint =>
-              <WaterpointMarker key={waterpoint.WATER_POINT_CODE} {...waterpoint} />
+              <WaterpointMarker center={waterpoint.position} key={waterpoint.WATER_POINT_CODE} />
             )}
           </BoundsMap>
           <SpinnerModal
               retry={load}
               state={this.state.waterpointsState} />
         </div>
-        <ChartsContainer>
-          charts for waterpoints...
+        <ChartsContainer
+            onToggle={toggleCharts}
+            state={this.state.layout.charts}>
           <MetricStatus metric="54.65" title="chart.title.functional" total="123456"/>
```

Comment:
I think this metric should be always visible. 
In here we should have new react component for charts